CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
```

Comment:
Maybe something in the Added section about adding examples with keycloak and OAuth2?

Reply:
done 👍